core/modules/user/user.admin.inc

```diff
@@ -500,6 +500,16 @@ function user_admin_permissions_submit($form, &$form_state) {
 function user_admin_roles($form, $form_state) {
   $system_config = config('system.core');
 
+  $form['roles_help_before'] = array(
+    '#type' => 'help',
+    '#markup' => t('Roles provide a way to organize people into groups that have similar duties. Each user account can be given one or more roles, and <a href="@permissions">permissions</a> can be granted to those roles (instead of individually to each user account).', array('@permissions' => url('admin/config/people/permissions'))),
+  );
+  $form['roles_help_after'] = array(
+    '#type' => 'help',
+    '#markup' => t('To make the permissions page easier to navigate, we recommended ordering roles from the least permissive (Anonymous) to the most permissive (usually Administrator).'),
```

Comment:
Help text usually speaks declarative (I think that's the right term?) rather than in first person.

```suggestion
    '#markup' => t('To make the permissions page easier to navigate, it is recommended to order roles from the least permissive (Anonymous) to the most permissive (usually Administrator).'),
```